res/css/_common.scss

```diff
@@ -21,6 +21,11 @@ limitations under the License.
 
 $hover-transition: 0.08s cubic-bezier(.46, .03, .52, .96); // quadratic
 
+$EventTile_e2e_state_indicator_width: 4px;
```

Comment:
Open for discussion.

I didn't find any variables yet. But since the values are required in different components I have put them here. If someone needs to change these values in the future he (hopefully) sees that they are used in multiple places.